DemoApp/app/screens/DataScreen.js

```diff
@@ -54,10 +54,26 @@ export default class DataScreen extends Component {
   async refreshToggle() {
     const dataEnabled = await Data.isEnabled();
     this.setState({ dataEnabled });
+    if (!this.state.loadingData) {
+      const documents = await this.listDocuments(this.state.partition);
+      this.setState({ documents, loadingData: false });
+    }
   }
 
-  setCreateDocModalVisible(visible) {
+  async setCreateDocModalVisible(visible) {
     this.setState({ createDocModalVisible: visible });
+    if (!visible) {
+      const documents = await this.listDocuments(this.state.partition);
+      this.setState({
+        docTtl: 60,
+        docId: '',
+        docType: '',
+        docKey: '',
+        docValue: '',
+        documents,
```

Comment:
For readability and consistency, I think the named parameter would be better.
```suggestion
        documents: documents,
```

Reply:
@dhei does `lint` agree with you?

Reply:
https://eslint.org/docs/rules/object-shorthand
You are right, eslint encourages using object shorthand in this case.